exercises/practice/resistor-color/runtests.jl

```diff
@@ -0,0 +1,32 @@
+using Test
+
+include("resistor-color.jl")
+
+@testset verbose = true "Resistor Color" begin
+    @testset "Black" begin
+        @test colorcode("black") == 0
+    end
+
+    @testset "White" begin
+        @test colorcode("white") == 9
+    end
+
+    @testset "Orange" begin
+        @test colorcode("orange") == 3
+    end
+
+    @testset "Colors" begin
+        @test colors() == [
+            "black",
+            "brown",
+            "red",
+            "orange",
+            "yellow",
+            "green",
+            "blue",
+            "violet",
+            "grey",
+            "white"
+        ]
```

Comment:
We should allow the student to return a variety of ordered collections (definitely vectors or tuples).

```suggestion
        expected_colors = [
            "black",
            "brown",
            "red",
            "orange",
            "yellow",
            "green",
            "blue",
            "violet",
            "grey",
            "white"
        ]
        @test all(colors() .== expected_colors)
```